barretenberg/cpp/src/barretenberg/protogalaxy/protogalaxy_prover_internal.hpp

```diff
@@ -160,16 +165,15 @@ template <class ProverInstances_> class ProtogalaxyProverInternal {
      *
      *
      */
-    static LegacyPolynomial<FF> compute_perturbator(std::shared_ptr<Instance> accumulator,
+    static LegacyPolynomial<FF> compute_perturbator(const std::shared_ptr<const Instance>& accumulator,
```

Comment:
I was just working with existing types, and I saw that Adam had made the perturbator a LegacyPolynomial and accepted that. Checked with him and he said it can be changed to a normal Polynomial, so I'll do that.